src/main/java/spark/Spark.java

```diff
@@ -36,11 +34,19 @@
  *
  * @author Per Wendel
  */
-public final class Spark extends SparkBase {
+public final class Spark {
```

Comment:
@jkwatson originally it was already final and I've tried to limit my changes to a bare minimum. I guess that after this PR is merged we could change if it gets traction.